hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/FileSystemConfigurations.java

```diff
@@ -81,5 +81,7 @@
   public static final String DEFAULT_FS_AZURE_USER_AGENT_PREFIX = EMPTY_STRING;
   public static final String DEFAULT_VALUE_UNKNOWN = "UNKNOWN";
 
+  public static final boolean DEFAULT_DELETE_CONSIDERED_IDEMPOTENT = true;
```

Comment:
Does it convey the right meaning by keeping DEFAULT prefix(as the same is not a configurable flag)?

Reply:
DEFAULT prefix is to highlight the default behaviour of these fields in ABFS driver. There do not mandate the need to be backed by user modifiable config in AbfsConfiguration. AbfsConfiguration is the consumer of these defaults.

Reply:
When you say this is to highlight the default behaviour, the assumption is there are non default behaviour to this feature, are there?

Reply:
As you said keeping the DEFAULT prefix to highlight the default behaviour it is assumed that there are non default behaviours associated with this feature, are there any?

Reply:
Definition of defaults need not be confused with what should be given an option to override using core-site configs. 

The behaviour of delete idempotency is being defined by default now, and the non default behaviour (where handling is absent) would be its not idempotent.